### ISSUE TITLE
Crash during export

### ISSUE THREAD

**Issue by Minh-Ly** [darktable-org/darktable#7610]
Describe the bug
I was exporting an image when darktable crashed.

Backtrace:
[darktable_bt_IICKW0.txt](https://github.com/darktable-org/darktable/files/5759387/darktable_bt_IICKW0.txt)

Platform (please complete the following information):
- Darktable Version: darktable master at 9ad9645
- OS: windows 10
- ~~OpenCL ON~~ OFF.
- NVidia 940MX, Driver 460.89

**Comment by aurelienpierre** [comment]
Trivial issue of division by zero, it seems.

**Comment by aurelienpierre** [comment]
Ok, not trivial issue. It's actually a bug with censorize.c IOP tiling since tiling changes merged today in master.

**Comment by parafin** [comment]
@TurboGit, can I ask how do you decide what goes into stable (3.4.x) branch? Previously only bugfixes went there, and this bug is IMHO is a good example why - by adding new functionality (censorize iop in this case) you also add new bugs. It also increases the amount of work for stable branch maintainer (you).
So what is conceptual difference nowadays between master and 3.4.x?

**Comment by TurboGit** [comment]
Not a lot of difference I make today. People have requested to have more like rolling releases. So I try to incorporate things that are mostly safe and should not crash whole dt. New IOP should be mostly safe (if you don't use them, no issue). Indeed that's quite a bit of work on my side.

Note that today I have a darktable-3.4.x branch with only very safe fixes. I made that to ensure that in case of hard crash or bad issue detected in 3.4.0 we can release a safe release.

I have queued many new things in the next branch that could go in 3.4.1 or 3.4.2 (if we have a need for a quick 3.4.1 release). The next branch is what we have on master and is tested by many users daily. All this should stabilize nicely. Note also that thanks to the integration test I'm far more confident about the stability of dt, and so all performances related changed have been added into next branch.

The goal is really to have rolling releases and addition of new feature regularly instead of many new features once a year. Hope this clarify my handling of the release.
